src/remoteFile.ts

```diff
@@ -1,26 +1,14 @@
 import uri2path from 'file-uri-to-path'
-import { GenericFilehandle, FilehandleOptions } from './filehandle'
+import { GenericFilehandle, FilehandleOptions, Stats, Fetcher, PolyfilledResponse } from './filehandle'
 import { LocalFile } from '.'
 
 const myGlobal = typeof window !== 'undefined' ? window : typeof self !== 'undefined' ? self : { fetch: undefined }
 
-/**
- * a fetch response object that might have some additional properties
- * that come from the underlying fetch implementation, such as the
- * `buffer` method on node-fetch responses.
- */
-interface PolyfilledResponse extends Response {
-  buffer: Function | void
-}
-
-interface Stats {
-  size: number
-}
 export default class RemoteFile implements GenericFilehandle {
-  private url: string
-  private _stat?: Stats
-  private fetch: Function
-  private baseOverrides: any = {}
+  public url: string
+  public _stat?: Stats
+  public fetch?: Fetcher
+  public baseOverrides: any = {}
```

Comment:
are these public just for the tests?

Reply:
No, they're public because my override of `getFetch` needs to be able to use them.

Reply:
Maybe protected then

Reply:
Wasn't familiar with 'protected', looks like exactly what I wanted. Thanks!